src/Sylius/Component/Review/Calculator/AverageRatingCalculator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Review\Calculator;
+
+use Sylius\Component\Review\Model\Reviewable;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class AverageRatingCalculator implements AverageRatingCalculatorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function calculate(Reviewable $reviewable)
```

Comment:
We will need more efficient way to calculate average rating. Calculating it by sum of ratings and amount of them should do the job.